scripts/prefsSetsDbUtils.js

```diff
@@ -0,0 +1,69 @@
+/*!
```

Comment:
Can we create a "script/shared" directory and move this script there? Also, add a comment in the script about who's sharing it and the goal of this sript.